Lib/test/libregrtest/run_workers.py

```diff
@@ -102,6 +102,9 @@ def __init__(self,
         super().__init__()
 
 
+_NOT_RUNNING = "<not running>"
```

Comment:
You can make it public, it's not a deep secret.

Reply:
I don't think that it needs to be public, since it is not used. If you prefer to have it as public - I will change it :)